app/models/counselor.rb

```diff
@@ -4,4 +4,6 @@ class Counselor < ApplicationRecord
   has_many :counselors_groups
   has_many :groups, through: :counselors_groups
 
+  validates :alias, presence: true
```

Comment:
Ouhhh yes yes trueeeee
I did this since the edit form was letting a blank alias go through.